cron/hourly/70-cancel-subscriptions-for-cancelled-orders.ts

```diff
@@ -28,24 +29,38 @@ const getHostFromOrder = async order => {
   return models.Collective.findByPk(transactions[0].HostCollectiveId);
 };
 
+const getOrderCancelationReason = (collective, order, orderHost) => {
+  if (!order.Tier) {
+    return `Order tier deleted`;
+  } else if (collective.deactivatedAt) {
+    return `@${collective.slug} archived their account`;
+  } else if (!collective.HostCollectiveId) {
+    return `@${collective.slug} was un-hosted`;
+  } else if (collective.HostCollectiveId !== orderHost.id) {
+    return `@${collective.slug} changed host`;
+  } else {
+    return `Order cancelled`;
+  }
+};
+
 /**
  * When archiving collectives. we need to make sure subscriptions managed externally are properly cancelled
  * by calling the right service method (PayPal). We do this asynchronously to properly deal with rate limiting and
  * performance constraints.
  */
 export async function run() {
   const orphanOrders = await models.Order.findAll({
+    where: { status: OrderStatuses.CANCELLED },
```

Comment:
Are we ready for this? It looks like some places where we're supposed to cancel the orders (such as `archiveCollective`) are not updating the order statuses yet.